### ISSUE TITLE
Error on server startup

### ISSUE THREAD

**Issue by dabou0** [Imanity-Software/ImanitySpigot3-git#37]
### Stack trace

```
https://paste.imanity.dev/M59G2A1
```

### Plugin list

I will disclose in private if needed

### Steps/models to reproduce (if known)

Just starting the server it gives this error

### ImanitySpigot3 version

2021.10.1 BUILD 2

### Agreements

- [X] I am running the latest version of ImanitySpigot3 available from
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have compared the issue inbetween TacoSpigot 1.8.8 and ImanitySpigot3 And only ImanitySpigot3 occured this issue.

**Comment by LeeGodSRC** [comment]
Hello, it seems like your plugin Box API is trying to teleport entity, please try to check from your plugins
```
at org.bukkit.craftbukkit.v1_8_R3.entity.CraftEntity.teleport(SourceFile:251)
at br.com.jddev.system.api.BoxAPI.lambda$loadTask$3(BoxAPI.java:182)
```

**Comment by dabou0** [comment]
https://code.imanity.dev/ObovSZ3

The error is giving only on imanityspigot3

**Comment by dabou0** [comment]
![image](https://user-images.githubusercontent.com/34607217/139970438-ee481b4d-d4ca-47e9-8a63-de83f1b1c8b8.png)
it gives this error on startup, tried regenerating the imanity folder, yet on restart it will give the same errors

**Comment by LeeGodSRC** [comment]
> https://code.imanity.dev/ObovSZ3
> 
> The error is giving only on imanityspigot3

Can you send us the plugin BoxAPI please? so we can check in our end

**Comment by LeeGodSRC** [comment]
> ![image](https://user-images.githubusercontent.com/34607217/139970438-ee481b4d-d4ca-47e9-8a63-de83f1b1c8b8.png) it gives this error on startup, tried regenerating the imanity folder, yet on restart it will give the same errors

This error has been fixed in **Rancha 2021.11.1 LTS BUILD 1**

**Comment by LeeGodSRC** [comment]
No response within last 5 days, please re-open whenever you can provide new information.
